src/pages/Dashboard/DashboardPageOnboard.module.scss

```diff
@@ -1,97 +1,86 @@
+@import "~@santiment-network/ui/mixins";
+
 .wrapper {
   position: relative;
   padding: 28px 24px 24px;
   margin-bottom: 70px;
+  background-color: var(--jungle-green-light);
+  background-image: url('./waves.svg');
 }
 
 .skip {
   cursor: pointer;
   position: absolute;
   top: 16px;
   right: 20px;
+
+  @include text('body-3');
+
+  &:hover {
+    color: var(--mirage);
+  }
+}
+
+.top {
+  display: flex;
+}
+
+.hand {
+  animation: rotation 2s infinite ease-in-out;
+  transform-origin: 100% 100%;
+}
+
+@keyframes rotation {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  50% {
+    transform: rotate(10deg);
+  }
+
+  100% {
+    transform: rotate(0deg);
+  }
 }
 
 .text {
-  text-align: center;
+  margin-left: 24px;
+  text-align: left;
 }
 
 .title {
-  font-size: 22px;
-  line-height: 30px;
   color: var(--mirage);
-  margin-bottom: 8px;
+  margin-bottom: 4px;
+
+  @include text('h4');
```

Comment:
Why do we use h4 in css, but not using html tag h4? And use div for all.
@DmitriVanGuard @haritonasty